corgidrp/detector.py

```diff
@@ -222,3 +222,26 @@ def make_detector_areas(detector_areas, areas=('image', 'prescan', 'prescan_reli
     for area in areas:
         detector_areas[area] = detector_area_mask(detector_areas, area=area)
     return detector_areas
+
+def get_rowreadtime_sec():
```

Comment:
I would add an argument of the time of observation (e.g., as an astropy.time object). This could be an optional argument for now. In the future, we can check on the time of observation to determine which rowreadtime to use.

Reply:
Should we use JD or ISO?

Reply:
In https://collaboration.ipac.caltech.edu/display/romancoronagraph/Header+Information, there's DATETIME, which is TBD UTC, and the other two keywords related to time are in TAI (SCTSRT, SCTEND). So, maybe UTC and  ISO 8601?

Reply:
I found 'DATE', '2024-02-10T00:17:48.172768' in tests/test_data/nonlin_sample.fits that Max created some time ago. I'll use then UTC and ISO8601

Reply:
@semaphoreP I am confused about your last comment below about using primitive types like None in the keyword of this function. I am not sure which comment said that. The only argument is the datetime. You asked: "I would add an argument of the time of observation (e.g., as an astropy.time object)" Do you mean not to use datetime=Time('2026-12-01 00:00:00', scale='utc') in the function definition? Can you clarify and provide an alternative if that's the case?